### PR TITLE
Feature/ocbm 159 set standard

### PR DESCRIPTION
**Pull request by Navin1196**
Completed UI for "set-standard Create" Table and implementation done with following components. 

1.created table UI with mock data.
2.UI completed for dropdown. 
3.wrapper UI done for whole create table component.
4.UI created for cancel and create button. 
5.UI done for checkbox with antd.

screenshot
image

figma screenshot
image

### CODE REVIEW

**Comment by chandran-kannan** [comment]
Hi @Navin1196 add figma design screen shots in PR

File: src/components/reusable/dropdown/index.tsx [hunk 1]
```diff
@@ -131,7 +132,9 @@ const Dropdown = ({
         </span>
       </div>
       {isOpen && (
-        <div className="absolute border-4 w-full my-3 border-white rounded-b-2xl rounded-l-2xl overflow-y-auto max-h-[277px] shadow-lg z-[99] bg-white">
+        <div
+          className={`absolute border-4 w-[300px] border-white rounded-b-2xl rounded-l-2xl max-h-[277px] shadow-lg z-[99] bg-white ${openClassName}`}
```
Comment (Jyshe) [inline src/components/reusable/dropdown/index.tsx]: did you check the other areas is it fine for the dropdown?

File: src/pages/login/LoginCard.tsx [hunk 2]
```diff
@@ -21,6 +21,7 @@ const LoginCard = () => {
     confirmPassword: '',
     reset: false,
   });
+  console.log(formData, 'form');
```
Comment (Jyshe) [inline src/pages/login/LoginCard.tsx]: remove console.log

File: src/pages/setStandards/newSetStandards.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.headerColor .ant-table-thead {
+  background-color: #cdcdcd;
+}
+.ant-pagination-item-active a {
```
Comment (Jyshe) [inline src/pages/setStandards/newSetStandards.css]: do the changes in index.css why external css for particular file?

**Review by chandran-kannan — APPROVED** [review]

**Pull request merged**